### ISSUE TITLE
reshape in RNN_dynamic

### ISSUE THREAD

**Issue by iLampard** [princewen/tensorflow_practice#17]
file 
https://github.com/princewen/tensorflow_practice/blob/master/nlp/RNN_dynamic_cell.py

line 75
logits = tf.reshape(tf.matmul(tf.reshape(rnn_outputs,[-1,**batch_size**]),W)+b,[batch_size,num_steps,num_classes])

此处reshape里面的batch_size是否应该改为state_size ?  因为例子里面batch_size和state_size都等于4，所以未报错，如果令batch_size=5，该行会抛出异常。

**Comment by princewen** [comment]
thanks
